app/models/game.rb

```diff
@@ -1,10 +1,35 @@
 class Game < ActiveRecord::Base
 
-  has_many :memberships
-  has_many :users, through: :memberships
+  validates :lobby, presence: true, uniqueness: true
 
-  def add_user(user)
-    memberships.create user: user
+  belongs_to :lobby
+
+  before_create :setup_game
+
+  CARD_ATTRIBUTES = {
+    symbol:  [ :diamond, :sguiggle, :oval ],
+    number:  [ :one, :two, :three ],
+    color:   [ :red, :green, :purple ],
+    shading: [ :solid, :striped, :open ]
+  }
+
+  def deck
```

Comment:
any way to avoid having conflicts between the activerecord method `deck` and this method? this can get pretty confusing pretty quickly for others.